### PR TITLE
Add if/unless bindings options.

### PR DESCRIPTION
**Pull request by Omicron7**
Useful to not send specifc changes to Global Registry.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,6 +149,10 @@ option is nil or empty. (default: `nil`) **[`:entity`]**
 * `:mdm_timeout`: Only pull mdm information at most once every `:mdm_timeout`. (default: `1.minute`)
 **[`:entity`]**
 
+* `:if`, `:unless`: Proc or Symbol, called to determine if the change should be sent (enqueue a worker) to Global
+Registry. Proc and Symbol will both receive the model for an entity, and the type and model for a relationship.
+**[`:entity`, `:relationship`]**
```
Comment (soberstadt) [inline README.md]: Could you include an example of using this option here?

Comment (Omicron7) [reply README.md]: I could show an example somewhere. That list of possible options probably isn't the best place. Was trying to keep that list as concise as possible.

Comment (soberstadt) [reply README.md]: One example here would be nice.

**Comment by Omicron7** [comment]
As written, it needs to be per class.
It seems counterintuitive to add an option to turn off the gem. I can't think of any other gems that do this. If you want the gem off, you don't initialize it or add config code.
With that in mind, it might be good for now to keep the monkeypatch in mpdx_api to disable workers completely for prod.

**Pull request merged**